.buildkite/forge.rayci.yml

```diff
@@ -13,3 +13,15 @@ steps:
   - name: oss-ci-base_ml
     wanda: ci/docker/base.ml.wanda.yaml
     depends_on: oss-ci-base_test
+
+  - name: corebuild
+    wanda: ci/docker/core.build.wanda.yaml
+    depends_on: oss-ci-base_build
+
+  - name: servebuild
+    wanda: ci/docker/serve.build.wanda.yaml
+    depends_on: oss-ci-base_build
+
+  - name: serverlessbuild
+    wanda: ci/docker/serverless.build.wanda.yaml
+    depends_on: oss-ci-base_build
```

Comment:
you actually diverged and did not use the `copy_everything` (though it is available already). this is awesome! great job!

so it seems that the `ci_build` image is actually not required for these tests to run?

could you split core and serve into two different PR's and also ask one or more eng from core / serve team to review?